config/sara/jurisdictions.yml

```diff
@@ -50,6 +50,7 @@
             required: false
             threshold_operator: 'Equal'
             group: 2
+            supplemental: true
```

Comment:
I think we might want to update this value for existing symptoms via the migration because the assessment that were created before and after this change look different

Reply:
Fixed in 743b019. Only updated the true values, since for all other values, they will default to `false`.